docs/doc/15-sql-functions/20-numeric-functions/index.md

```diff
@@ -0,0 +1,40 @@
+| <a name='pow'></a>[POW(x, y)](#pow)       | Returns the value of x to the power of y.                                                                                                                                                                                                                                                                                                                      | POW(-2, 2)           | 4                    |
+| <a name='power'></a>[POWER(x, y)](#power)     | Alias of POW. Returns the value of x to the power of y.                                                                                                                                                                                                                                                                                                        | POWER( -2, 2 )       | 4                    |
+| <a name='radians'></a>[RADIANS(x)](#radians) | Returns the argument X, converted from degrees to radians.                                                                                                                                                                                                                                                                                                     | RADIANS ( 90 )       | 1.5707963267948966   |
+| <a name='rand'></a>[RAND()](#rand)             | Returns a random floating-point value v in the range 0 <= v < 1.0. To obtain a random integer R in the range i <= R < j, use the expression FLOOR(i + RAND() * (j − i)).                                                                                                                                                                                       | RAND ( )             | various              |
+| <a name='randn'></a>[RAND(n)](#randn)            | Returns a random floating-point value v in the range 0 <= v < 1.0. To obtain a random integer R in the range i <= R < j, use the expression FLOOR(i + RAND() * (j − i)).  Argument n is used as the seed value. For equal argument values, RAND(n) returns the same value each time , and thus produces a repeatable sequence of column values (last example). | RAND ( 1 )           | various              |
+| <a name='round'></a>[ROUND(x, d)](#round)     | Rounds the argument x to d decimal places. The rounding algorithm depends on the data type of x. d defaults to 0 if not specified. d can be negative to cause d digits left of the decimal point of the value x to become zero. The maximum absolute value for d is 30; any digits in excess of 30 (or -30) are truncated.                                     | ROUND ( 0.123, 2 )   | 0.12                 |
```

Comment:
Pls don't put html inside a markdown, it's hard to write and maintain.

[Duckdb] do this like https://raw.githubusercontent.com/duckdb/duckdb-web/master/docs/sql/functions/numeric.md

Reply:
yeah, I agree. But @BohuTANG  want anchor point for each function, so that we can easily share the function link. Markdown don't support this as a default behavior. This is kind of hacky, but this is the only way I aware of.

Reply:
It's already a single page,  `anchor point` can be used via `Ctrl + F`.   Adding HTML inside markdown makes things complex.

Reply:
That's OK with me.

Reply:
SG! I'll just put it as plain table, thanks for the input!